src/buffers/pre_boot_buffer.c

```diff
@@ -0,0 +1,4 @@
+#include "buffers.h"
+
+// Dummy, the size of this symbol will adjust itself based on the spec
```

Comment:
Not a fan of the comment, the symbol itself will stay at a size of 1 element, maybe you could instead say something like "marks the start of xxx space whose actual size depends on the spec"